content/solutions/altinn-studio/functional/text/_index.md

```diff
@@ -41,3 +41,117 @@ Where the first param is the language key and the second is the language element
 - Keys are written in lowercase with underscores (example_key)
 
 {{<figure src="oversetting.png?width=1000" title="Editor for oversetting av tekster">}}
+
+# Language best practice
```

Comment:
Top level heading is defined in front matter.

Reply:
Fixed.